packages/sdk/src/core/ConfigSchema.ts

```diff
@@ -26,6 +26,10 @@ export class ConfigSchema {
         // }
       });
   }
+
+  static merge(...schemas: ConfigSchema[]): ConfigSchema {
+    return new ConfigSchema(_.merge({}, ...schemas.map((schema) => schema.keys)));
+  }
```

Comment:
I suggest we keep this logic contained within the common package for the glob patterns feature. The internal SDK should be as lean as possible to keep the architecture of the project as straightforward as possible.

Reply:
Makes sense :) I'll look to make this change sometime this evening.